docs/user-guide/global-configurations/okta.md

```diff
@@ -0,0 +1,63 @@
+# Example - Okta SSO
+
+## Prerequisites
+
+A verified account on [Okta](https://www.okta.com/). Okta activates your account only if email verification is successful.
+
+Here's a reference guide to set up your Okta org and application: [Link](https://developer.okta.com/docs/guides/oie-embedded-common-org-setup/go/main/)
+
+## Tutorial
+
+{% embed url="https://www.youtube.com/watch?v=_2f-5WGmQ4Y" caption="Okta App Setup" %}
+
+## Steps on Okta Admin Console
+
+Once your Okta org is set up, create an app integration on Okta to get a Client ID and Client Secret.
+
+1. In the Admin Console, go to **Applications** → **Applications**.
+
+2. Click **Create App Integration**.
+
+3. Select **OIDC - OpenID Connect** as the **Sign-in method**.
+
+{% hint style="info" %}
+OIDC stands for OpenID Connect. [Click here](https://www.okta.com/openid-connect/) to read more.
+{% endhint %}
+
+4. Select an application type (Web, Single Page, or Native), and click **Next**.
+
+5. On the **App Integration** page:
+    * Give a name to your application.
+    * Select the **Interaction Code** and **Refresh Token** checkbox.
+    * Now go to Devtron's Global Configurations → SSO Login Services → OIDC.
+    * Copy the redirect URI given in the helper text (might look like: https://xxx.xxx.xxx/xxx/callback).
+    * Return to the Okta screen, and remove the prefilled value in **Sign-in redirect URIs**.
+    * Paste the copied URI in **Sign-in redirect URIs**.
+    * Click **Save**.
+
+6. On the **General** tab:
+    * Note the **Client ID** value.
+    * Click the **Edit** option.
+    * In Client Authentication, choose **Client Secret**.
+    * Click **Save**.
+    * Click **Generate new secret**.
+    * Note the **Client Secret** value.
+
+
+## Steps on Devtron
+
+1. Go to the Global Configurations → SSO Login Services → OIDC.
+2. In the **URL** field, enter the Devtron application URL (a valid https link) where it is hosted.
+3. In `config`, provide the `clientID` and `clientSecret` of the app integration you created on Okta.
+4. Add a key `insecureSkipEmailVerified: true`.
```

Comment:
If this is always required, why don't we have it prefilled in the configuration? Why user need to provide it manually?

Reply:
not always need, could be different for other type of oidc sso.

Reply:
we can add a note that this key is only required for okta oidc sso. cc: @ashokdevtron

Reply:
We can add this [doc link](https://dexidp.io/docs/connectors/oidc/) too, saying here user may refer for available configuration options, something around this.